### ISSUE TITLE
Collision stiffness in collision penalty?

### ISSUE THREAD

**Issue by adam-lau709** [isantesteban/snug#17]
Hello, 

Thank you very much for sharing the great work and this repository. However, for the collision penalty in this repo, the collision stiffness seems to be missing:
```python
def collision_penalty(va, vb, nb, eps=1e-3):
    batch_size = tf.cast(tf.shape(va)[0], va.dtype)

    closest_vertices = NearestNeighbour(dtype=va.dtype)(va, vb)
    vb = tf.gather(vb, closest_vertices, batch_dims=1)
    nb = tf.gather(nb, closest_vertices, batch_dims=1)

    distance = tf.reduce_sum(nb*(va - vb), axis=-1) 
    interpenetration = tf.maximum(eps - distance, 0)

    return tf.reduce_sum(interpenetration**3) / batch_size
```
Should the last line be `return tf.reduce_sum(250*interpenetration**3) / batch_size` instead? 

Also, in issue #11 you have provided the values of the energies during the training process, and the untrained values are:
> full loss: 392.6, bending: 1.273, collisions: 8.24e-4, gravity: 0.053, inertia: 0.046, stretch: 391

I am a bit confused about the initial collision value. How could it be so low for the untrained model? Since the garment displacements are random at the beginning, there should be a lot of collision with the body isn't it? In my implementation, the initial collision penalty is around 20 to 30 and I was using the same code as provided in this repo, except that I multiplied the collision stiffness. 

Maybe I have missed some important parts. Any hints will be appreciated!

**Comment by alisherakb** [comment]
Hi @Aorrr ,

You're correct regarding the missing collision penalty value, the author confirmed it to me

**Comment by adam-lau709** [comment]
Hi Alisher,

Thank you for sharing this information. As for the huge collision penalty, I just realise that I forgot to take the translation into account.
